### DISCUSSION TITLE
get form loading state

### DISCUSSION THREAD

**Discussion by azrlucky** [optimajet/formengine#36] [Q&A] [answered]
Hi Team,
Can you please guide me to get the property or callback when the form is completely loaded? we have a very lengthy form and it takes quite a bit to load in designer as well as viewer. we want to show a loader by the time the form is loading but unsure about any trigger/callback to know when the form loaded so we can stop the loader.

**Comment by sergeythrees** [comment, marked as answer]
> Hi Team, Can you please guide me to get the property or callback when the form is completely loaded? we have a very lengthy form and it takes quite a bit to load in designer as well as viewer. we want to show a loader by the time the form is loading but unsure about any trigger/callback to know when the form loaded so we can stop the loader.

Hi, @azrlucky 
we will think about this functionality and let you know when we decide how to implement it.
